octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,13 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        System.out.println("Sum of firstNumber and secondNumber is: " + add(10, 20));
+    }
+
+    private static int add(int firstNumber, int secondNumber) {
+        return firstNumber + secondNumber;
+    }
+
```

Comment:
[Severity: Minor] The code is well-structured and easy to follow, which is great!
Keep up the good work on maintaining clarity in your code structure.
Rationale: Clear structure is essential for readability and maintainability in programming.
Reference: https://www.oracle.com/java/technologies/javase/codeconventions-contents.html